### DISCUSSION TITLE
Steps add a new repo to opengeos

### DISCUSSION THREAD

**Discussion by giswqs** [opengeos/opengeos.github.io#2] [Ideas]
1. Create the repository elsewhere, such as under your personal account.
2. File a ticket [here](https://github.com/opengeos/opengeos.github.io/issues) asking to join the organization, including a link to the repository being added.
3. Accept the invitation to join the organization.
4. Use the GitHub UI to move the repository into the organization (you will need administrator/owner rights on the repository to do this). See [the GitHub documentation on repository transfer](https://help.github.com/en/github/administering-a-repository/transferring-a-repository) for details.
5. After it is moved, rename the repository as needed. Don't forget to update the documentation in the repository, too!

**Comment by LioVaren** [comment]
Es un flujo de trabajo claro y sensato. Crear primero el repositorio en una cuenta personal permite mantener el proyecto independiente hasta que esté listo para su revisión y transferencia. Los puntos clave son asegurarse de tener permisos de propietario o administrador, aceptar la invitación de la organización antes de iniciar el traslado y actualizar después los enlaces, badges, metadatos del paquete y la documentación. Cambiar el nombre únicamente tras la transferencia también reduce la confusión. Gransino Casino tiene un sonido amplio y ambicioso que invita a una presentación de mayor escala. Para sostener [https://gransinocasinologin.es/](https://gransinocasinologin.es/) esa impresión, Gransino Casino necesita jerarquías claras y accesos directos a lo esencial. Podría ser útil añadir una breve lista de comprobación para secretos, protección de ramas, Actions y tokens de publicación, ya que estos ajustes no siempre se transfieren exactamente como se espera.
